app/src/main/java/org/mozilla/fenix/wallpapers/WallpaperFileManager.kt

```diff
@@ -36,27 +42,21 @@ class WallpaperFileManager(
     }
 
     private fun getAllLocalWallpaperPaths(name: String): List<String> =
-        listOf("landscape", "portrait").flatMap { orientation ->
-            listOf("light", "dark").map { theme ->
-                Wallpaper.getBaseLocalPath(orientation, theme, name)
-            }
+        Wallpaper.ImageType.values().map { orientation ->
+            getLocalPath(orientation, name)
```

Comment:
is it only the web path/URL that needs the image type name as lowercase?

Reply:
The image type is also lowercased inline in `getLocalPath`. That function will just be referenced from multiple places, whereas the `remotePath` is only ever used from the downloader.